h/security/policy/_api_cookie.py

```diff
@@ -28,7 +30,18 @@ def handles(request: Request) -> bool:
         ) in COOKIE_AUTHENTICATABLE_API_REQUESTS
 
     def identity(self, request: Request) -> Identity | None:
+        try:
+            check_csrf_origin(request)
```

Comment:
I would suggest adding a note here about why we are requiring the CSRF token even though `SameSite=Strict` actually protects against the "cross-site" aspect problem. The main advantage of a CSRF token over a `SameSite=Strict` cookie in a modern browser is that it isn't set if a user follows a link in user-generated content on the same site (eg. a link in an annotation). CSRF tokens used to have a benefit they worked even in browsers that didn't support `SameSite=Strict`, but that support has been widespread for a long time now.

If we could rely on [`Sec-Fetch-*`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Sec-Fetch-Mode) headers in future, we could perhaps avoid the need for this.

Reply:
Added